gradle/root/java6-compatibility.gradle

```diff
@@ -3,10 +3,12 @@
 if (project.hasProperty('checkJava6Compatibility') && !System.getenv("SKIP_RELEASE")) {
     //if we're skipping release, let's also skip checking compatibility (faster builds)
     allprojects { p ->
-        plugins.withId('java') {
-            p.apply plugin: 'ru.vyarus.animalsniffer'
-            p.dependencies {
-                signature 'org.codehaus.mojo.signature:java16:1.0@signature'
+        if (p.name != 'junit5') {
+            plugins.withId('java') {
+                p.apply plugin: 'ru.vyarus.animalsniffer'
+                p.dependencies {
+                    signature 'org.codehaus.mojo.signature:java16:1.0@signature'
+                }
```

Comment:
I am not really sure what you mean with baseline. We configure the Java 1.8 compatibility in the subproject:

```gradle
sourceCompatibility = 1.8
targetCompatibility = 1.8
```

Could you clarify your comment?

Reply:
Source and target compatibility make sure you only use Java 8 language features and compile to Java 8 byte code, respectively. The animalsniffer plugin checks that you don't use any APIs that are not present in whatever baseline you pick. For example, if you would use an API that was introduced in Java 9, and the build runs on Java 9, it would compile without any issues. Users on Java 8 would however get a runtime exception. For Java 8, there's `org.codehaus.mojo.signature:java18:1.0`.

Reply:
Oh I see, yes good call. Done :+1: